app/presenters/claim_state_transition_presenter.rb

```diff
@@ -15,6 +15,20 @@ def audit_users
     format(sentence, author: transition.author.name, subject: transition.subject&.name)
   end
 
+  def reason_header
+    "#{'Reason'.pluralize(transition.reason.size)} provided:"
```

Comment:
**[not a blocker] ** This seems to be trying to achieve the same as this: https://robots.thoughtbot.com/pluralizing-rails-i18n-translations. Consider using the Rails one instead.